_data/2016-17/national/departments/science-and-technology.yaml

```diff
@@ -109,7 +109,7 @@ budget_actual:
       financial_year: 2014-15
       phase: Audit Outcome
     real:
-    - amount: 8737160692
+    - amount: 8737160687
```

Comment:
what's the correct number in the spreadsheet?

Reply:
![2015-ene-science-and-tech](https://user-images.githubusercontent.com/7869992/54415508-2274cf80-4705-11e9-82e9-aba0117a2005.PNG)

It looks like the new value is the correct value, according to the 2015 ENE dataset.